run

```diff
@@ -1061,6 +1063,83 @@ addMode('geo880', 'Semantic parsing on the geo880 dataset', lambda { |e| l(
 nil) })
 
 ############################################################
+# {2016-07-02} [Sida Wang]: interactive semantic parsing
+addMode('interactive', 'interactive semantic parsing in a blocksworld', lambda { |e| l(
```

Comment:
separate run scripts is good, especially if we can use the common setup code for wrapping under fig/codalab etc.